pkgs/development/ocaml-modules/bap/default.nix

```diff
@@ -36,6 +41,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ ocaml findlib ocamlbuild ocaml_oasis
                   llvm ppx_tools_versioned
+		  z3
```

Comment:
Nit: This indentation looks off.

Reply:
Indeed, there were some unwanted tabs there.